boards/arduino-uno/src/lib.rs

```diff
@@ -68,7 +68,10 @@ pub use crate::pac::Peripherals;
 mod pins;
 pub use crate::pins::*;
 
-pub use crate::hal::prelude;
+pub mod prelude {
+    pub use crate::hal::prelude::*;
+    pub use crate::hal::usart::BaudrateExt as _;
```

Comment:
I think (but we need to test!) for the uno we need the special implementation:

```suggestion
    pub use crate::hal::usart::BaudrateArduinoExt as _;
```